### PR TITLE
Refactored logging statements to use lazy % formatting

### PR DESCRIPTION
**Pull request by skynette**
Refactored most if not all logging statements to use % lazy format.
The goal of this PR is to refactor some of the logging statements to use the % lazy style of formatting

for example refactoring
logger.info(f"Fetching {events_url}") 

becomes

logger.info("Analyzing {}...".format(bug_fixing_commit["rev"]))

This PR is part of the fix for #3323.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,6 @@ node_modules/
 *.log
 # Desktop Service Store
 *.DS_Store
+
+# virtual environments
+env/
```
Comment (suhaibmujahid) [inline .gitignore:55]: Could you add drop unrelate changes?

To ignore `env`,  you could add a `.gitignore` file inside `env`:
```
echo "*" > env/.gitignore
```

Or you could add `env/` to a global git ignore file.

**Review by suhaibmujahid — CHANGES_REQUESTED** [review]
Thank you, @skynette! Could you please fix the lint errors?

To avoid lint failures, it is recommended to setup pre-commit: https://github.com/mozilla/bugbug#auto-formatting

File: bugbug/repository.py [hunk 2]
```diff
@@ -963,7 +963,7 @@ def _hg_log(revs: list[bytes], branch: str = "tip") -> tuple[Commit, ...]:
 
 
 def get_revs(hg, rev_start=0, rev_end="tip"):
-    logger.info(f"Getting revs from {rev_start} to {rev_end}...")
+    logger.info("Getting revs from %d to %s...", rev_start, rev_end)
```
Comment (suhaibmujahid) [inline bugbug/repository.py:966]:
```suggestion
    logger.info("Getting revs from %s to %s...", rev_start, rev_end)
```

**Comment by skynette** [comment]
Okay I will attend to these thanks for feedback

File: .gitignore [hunk 3]
```diff
@@ -49,4 +49,4 @@ node_modules/
 # Logs
 *.log
 # Desktop Service Store
-*.DS_Store
+*.DS_Store
```
Comment (suhaibmujahid) [inline .gitignore:52]: Could you pleas drop unrelated changes?

Comment (suhaibmujahid) [reply .gitignore:52]: (put back the empty line at the end of the file)

Comment (skynette) [reply .gitignore:52]: okay got it

**Review by suhaibmujahid — APPROVED** [review]
LGTM!
